test/test-utils.js

```diff
@@ -0,0 +1,90 @@
+/*jshint expr: true*/
+"use strict";
+var chai = require("chai");
+var expect = chai.expect;
+chai.config.includeStack = true;
+var utils = require("../utils");
+
+describe("Utilities testing", function() {
```

Comment:
In BDD you describe what things are or "how they behave", instead of what you're doing. So this should just be `Utils`.